be/src/main/java/com/codesquad/issuetracker/api/issue/controller/IssueController.java

```diff
@@ -0,0 +1,29 @@
+package com.codesquad.issuetracker.api.issue.controller;
+
+import com.codesquad.issuetracker.api.issue.dto.IssueCreateRequest;
+import com.codesquad.issuetracker.api.issue.service.IssueService;
+import java.util.Map;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+@Controller
+@RequiredArgsConstructor
+public class IssueController {
+
+    private final IssueService issueService;
+
+    @PostMapping("/api/{organizationTitle}/issues")
+    public ResponseEntity<Map<String, Long>> create(@RequestBody IssueCreateRequest issueCreateRequest,
+                                                    @PathVariable String organizationTitle) {
+        // TODO: 로그인 관련 처리 필요
+        Long issueId = issueService.create(organizationTitle, issueCreateRequest);
```

Comment:
```suggestion
        Long issueId = issueService.create(organizationTitle, issueCreateRequest,userId);
```
- userId로 파라미터로 넘으면 어떨까요?  아니면 issueCreateRequest에서 userId를 set하고 Service에서 다시 get하는 작업이 있을 수 있을 것 같아요.